Documentation/CRDs/Object-Storage/ceph-object-multisite-crd.md

```diff
@@ -84,6 +84,8 @@ spec:
     erasureCoded:
       dataChunks: 2
       codingChunks: 1
+  customEndpoints:
```

Comment:
Looks like there is an extra space so it looks like it's a setting under `dataPool`, but it's a top level setting, right?

Reply:
Yes it top level setting, in my vim editor it is showing as same level as `dataPool`, not sure why it is list like that. Can you please give it as suggestion, so that I can directly commit to this PR

Reply:
It should be using spaces. Do you have a tab instead? If it's already spaces, just remove one space.